docs/setup/install-with-binary.md

```diff
@@ -143,7 +143,7 @@ make
 
 The compiled kubeedge binaries will be put to `_output/local/bin` directory.
 
-### Deploy demo on edge nodes
+## Deploy demo on edge nodes
```

Comment:
i believe that this indent is wrong since `application demo` should not included under `keadm join` section.